tasks/kernel_matrix_testing/download.py

```diff
@@ -1,33 +1,25 @@
-import json
 import os
 import platform
 import tempfile
+from typing import List
 
+from invoke.context import Context
+
+from tasks.kernel_matrix_testing.platforms import get_platforms
 from tasks.kernel_matrix_testing.tool import Exit, debug, info, warn
+from tasks.kernel_matrix_testing.vars import arch_mapping
+from tasks.kernel_matrix_testing.vmconfig import get_vmconfig_template
 
 try:
     import requests
 except ImportError:
     requests = None
 
-platforms_file = "test/new-e2e/system-probe/config/platforms.json"
-
-
-def get_vmconfig_file(template="system-probe"):
-    return f"test/new-e2e/system-probe/config/vmconfig-{template}.json"
 
+def requires_update(url_base: str, rootfs_dir: str, image: str, branch: str):
```

Comment:
Why not use `Optional[Union[PathLike, str]]` for `rootfs_dir` like above?

Reply:
I missed this one, changed it!